src/makerspace/views.py

```diff
@@ -67,3 +70,48 @@ class EquipmentDeleteView(PermissionRequiredMixin, PreventGetRequestsMixin, Dele
     permission_required = ('makerspace.delete_equipment',)
     model = Equipment
     success_url = reverse_lazy('admin_equipment_list')
+
+
+class StatisticsView(TemplateView):
+    template_name = 'makerspace/statistics.html'
+    get_time_in_hours = 3600000000  # number the reservation length needs to be divided by to get the length in hours
+    filter_reservations = Q(reservations__special=False) & Q(reservations__event__isnull=True)
+
+    reservations = Reservation.objects.prefetch_related(Prefetch('machine', queryset=Machine.objects.
```

Comment:
The exprassions for reservations (line 80 – 82) sewingmachines (line 84 – 87) and printers (line 89 – 92) are extremely long and ugly. Would it be possible to refactor these, to make them more readable?